setup.py

```diff
@@ -132,23 +132,20 @@ def lib_ext_kwargs(pc, prefix_env_var, lib_name, lib_pkg_name, pc_version, lib_s
             f"or ensure {lib_pkg_name}.pc is in PKG_CONFIG_PATH."
         )
 
-    crypto_extra_objects = []
     if is_win32:
         crypto_ext_lib = lib_ext_kwargs(pc, "BORG_OPENSSL_PREFIX", "libcrypto", "libcrypto", ">=1.1.1", lib_subdir="")
     elif is_openbsd:
         # Use openssl (not libressl) because we need AES-OCB via EVP api. Link
         # it statically to avoid conflicting with shared libcrypto from the base
-        # OS pulled in via dependencies.
-        crypto_ext_lib = {"include_dirs": ["/usr/local/include/eopenssl30"]}
-        crypto_extra_objects += ["/usr/local/lib/eopenssl30/libcrypto.a"]
+        # OS pulled in via dependencies. Make crypto paths configurable.
+        include_dirs = [os.environ.get("BORG_OPENSSL_INC", "/usr/local/include/eopenssl30")]
+        extra_objects = [os.path.join(os.environ.get("BORG_OPENSSL_LIB", "/usr/local/lib/eopenssl30"), "libcrypto.a")]
+        crypto_ext_lib = {"include_dirs": include_dirs, "extra_objects": extra_objects}
```

Comment:
I recently had a look at this, whether the code used for the other platforms could do this - and the main problem I found is that it needs that libcrypto.a here in extra_objects.

But I thought:
- it could use BORG_OPENSSL_PREFIX=/usr/local
- a new BORG_OPENSSL_NAME=eopenssl30 to be able to vary that part of the path.
- guess everything else is fixed?

Reply:
@bket seen this ^ ?

Reply:
@ThomasWaldmann, yes, I saw your comments, and updated the PR accordingly.